src/components/FolderOwner.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { getMockFolder } from '../api';
+import useAsync from '../hooks/useAsync';
+import './FolderOwner.css';
+
+function FolderOwner() {
+  const [, , getOwnerInfo] = useAsync(getMockFolder);
+  const [owner, setOwner] = useState([]);
+  const [star, setStar] = useState('');
+
+  const getOwner = async () => {
+    const result = await getOwnerInfo();
+    if (!result) return;
+
+    const {
+      folder: { owner },
+      folder: { name },
+    } = result;
```

Comment:
result 의 folder 객체에서 owner, name 을 꺼내오고 싶은거라면 아래처럼 해도 됩니다!

```suggestion
      folder: { owner, name },
    } = result;
```